app/lib/three_scale/policies/schemas/apicast-policy-v1.schema.json

```diff
@@ -1,11 +1,11 @@
 {
-  "$id": "http://apicast.io/policy-v1/schema#manifest",
+  "$id": "http://apicast.io/policy-v1/schema",
   "type": "object",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "definitions": {
     "schema": {
       "$id": "#/definitions/schema",
-      "$ref": "http://json-schema.org/draft-07/schema#",
+      "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
This `$ref` here also was problematic with `json_schemer`, it causes the following error:

`JSONSchemer::UnknownRef "http://json-schema.org/draft-07/schema"`.

I actually don't know whether it is supposed to work, but in fact, I assume that `$schema` is what was meant here, because this is the schema that the "configuration" property of the policy must follow.